data/xp-pen-artist10s.tablet

```diff
@@ -0,0 +1,30 @@
+# XP-Pen
+# Artist 10S (1st generation)
+#
+# sysinfo.2kd91mftwQ
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/354
+
+[Device]
+Name=UGEE 10.1 Tablet Monitor
+ModelName=
+DeviceMatch=usb:5543:004a:UGEE 10.1 Tablet Monitor;usb:5543:004a:UGEE 10.1 Tablet Monitor Pad;usb:5543:004a:UGEE 10.1 Tablet Monitor Mouse
+PairedIDs=
+Class=Bamboo
+Width=9
+Height=5
+IntegratedIn=
+Layout=xp-pen-artist10s.svg
+Styli=@generic-no-eraser;
+
+[Features]
+Stylus=true
+Reversible=true
+Touch=false
+TouchSwitch=false
+Ring=false
+Ring2=false
+NumStrips=1
```

Comment:
`Bail out! ERROR:../test/test-tablet-svg-validity.c:136:check_touchstrip: assertion failed: (node != NULL)`

CI is complaining because the svg doesn't have a touch strip defined. Looks like the `data/xp-pen-artist12.tablet` file has one strip defined too so you can probably peek at that svg for guidance.

Reply:
Ah, I see. Does this NumStrip means touch strip? This tablet (10s) has 6 buttons only, and they are sending event codes BTN_0 ... BTN_5 that's why I thought this is a NumStrip. Should I just remove it from the .tablet file then (NumStrip=0 is the default) ? See my commit.

Reply:
yep, `NumStrips` defines the number of touch strips. If yours doesn't have any of those then setting it to zero (or leaving it out) is enough.